BricksAndHearts.UnitTests/ServiceTests/Property/PropertyServiceTests.cs

```diff
@@ -147,7 +147,8 @@ public async void UpdateProperty_UpdatesProperty()
         property.AcceptsBenefits.Should().BeFalse();
         context.Properties.Count().Should().Be(propertiesBeforeCount);
     }
-
+    
+    /* This logic now occurs in the View Model, causing the test to fail

```

Comment:
Instead of commenting we should probably remove

Reply:
I've rewritten it to test the logic in the View Model (I thought it would be more complicated than it was)